cli/xxhsum.c

```diff
@@ -59,7 +59,8 @@
 
 static unsigned XSUM_isLittleEndian(void)
 {
-    const union { XSUM_U32 u; XSUM_U8 c[4]; } one = { 1 };   /* don't use static: performance detrimental  */
+    union { XSUM_U32 u; XSUM_U8 c[4]; } one;   /* don't use static: performance detrimental  */
```

Comment:
You might want to check that the compiler still generates the same code for that variant.
Not concerned about correctness : it should be fine in both cases.
But we want to be sure that performance is also not impacted.